### PR TITLE
replace `torch.Tensor` by `torch.tensor` in docstring

### LINKED ISSUE #2527: Replace all `torch.Tensor` by `torch.tensor`
## 📚 Documentation

@sdesrozis  it is incorrect to create tensors with `torch.Tensor` and should be replaced by `torch.tensor`. 

https://github.com/pytorch/ignite/blob/3b821e27ec41fffbbe5e43658798040f7024ba0b/ignite/metrics/accuracy.py#L125-L134

This way we wont need neither cast to `.long` : `y_true = torch.Tensor([2, 0, 2, 1, 0, 1]).long()`

https://github.com/pytorch/ignite/blob/3b821e27ec41fffbbe5e43658798040f7024ba0b/ignite/metrics/accuracy.py#L146

### PR DESCRIPTION
**Pull request by sdesrozis**
Fixes #2527

Description:

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

**Review by vfdev-5 — APPROVED** [review]
LGTM
